solr/core/src/test/org/apache/solr/cloud/TestCloudRecovery.java

```diff
@@ -49,7 +49,6 @@
 
 public class TestCloudRecovery extends SolrCloudTestCase {
 
-  private static final String COLLECTION = "collection1";
```

Comment:
Could change to the following to reduce size of diff?:

```
private static final String COLLECTION = DEFAULT_TEST_COLLECTION_NAME;
```

Reply:
i started acoutally out with that...    And my thought was that the value of this entire PR is directly based on how much we get every test in every file to using the same name...    I really didn't want to go through by hand and fix all of these... ;-).   My thought is that by using the label DEFAULT_TEST_COLLECTION_NAME everywhere, it signals to you that you don't need to think about what the `COLLECTION` or `collection` variable means..   and that if you see either of those variables, then it presumably matters for some reason to that test....